app/frontend/src/aquifers/components/View.vue

```diff
@@ -163,9 +178,15 @@ export default {
       'fileUploadSuccess',
       'fileUploadFail'
     ]),
+    bySection (resources, section) {
+      return (resources || []).filter(function(resource){
+        return resource.section_id === section.id
+      })
+    },
     handleSaveSuccess (response) {
       this.fetch()
       this.navigateToView()
+      
```

Comment:
`✘  http://eslint.org/docs/rules/no-trailing-spaces           Trailing spaces not allowed`